tests/fe/templates_test.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+import os
+from dataclasses import fields, is_dataclass
+from typing import TYPE_CHECKING
+
+from jinja2.meta import find_undeclared_variables
+
+import grouper.fe
```

Comment:
Does this take care of importing all the template classes so `BaseTemplate.__subclasses__()` finds them? I never remember details of Python import behavior

Reply:
It relies on the fact that they're all defined in the same file. I'll add an explicit comment to that effect to `grouper.fe.templates`.